.claude/commands/create_pr.md

```
@@ -0,0 +1,40 @@
+
+## 실행 순서
+
+1. 현재 브랜치와 변경사항 확인
+   - `git branch --show-current`
+   - `git log dev..HEAD --oneline`
+   - `git diff dev...HEAD --stat`
+
+2. 변경사항을 분석해서 PR 제목 생성
+   - 형식: `[type]: 한 줄 요약`
+   - type 선택 기준:
+     | type | 사용 시점 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)